docs/2.guide/1.concepts/1.auto-imports.md

```diff
@@ -140,9 +140,30 @@ export default defineNuxtConfig({
   }
 })
 ```
-
 This will disable auto-imports completely but it's still possible to use [explicit imports](#explicit-imports) from `#imports`.
 
+### Partially Disabling Auto-imports
+
+If you want framework-specific functions like `ref` to remain auto-imported but wish to disable auto-imports for your own code (e.g., custom composables), you can set the `imports.scan` option to `false` in your `nuxt.config.ts` file:
+
+```ts
+export default defineNuxtConfig({
+  imports: {
+    scan: false
+  }
+})
+```
+
+With this configuration:
+- Framework functions like `ref`, `computed`, or `watch` will still work without needing manual imports.
+- Custom code, such as composables, will need to be manually imported in your files.
+
+::warning
+**Caution:** This setup has certain limitations:
+- If you structure your project to include an "abstract layer" which extends from a "core layer" (which is an npm package), you’ll need to import composables from the npm package similarly to third-party packages (e.g., `vue-i18n`) into your "abstract layer".
```

Comment:
```suggestion
- If you structure your project with layers, you will need to explicitly import the composables from each layer, rather than relying on auto-imports.
```